scripts/fix-encoding.py

```
@@ -0,0 +1,116 @@
+Automated UTF-8 encoding fixer for C# documentation.
+Fixes common encoding corruption issues where Edit tools convert proper UTF-8 to replacement characters.
+"""
+
+import sys
+import os
+import re
+from pathlib import Path
+
+def fix_encoding(content):
+    """Fix all common UTF-8 encoding corruptions."""
+    original = content
```

Ruff (0.14.4)

Local variable `original` is assigned to but never used

Remove assignment to unused variable `original`

(F841)